### DISCUSSION TITLE
Dart

### DISCUSSION THREAD

**Discussion by sthakrishnakumar** [community/community#19971] [Other Feature Feedback, Questions, & Ideas] [answered]
What is double dot in Dart?

**Comment by surajadkhari** [comment, marked as answer]
.. is known as cascade notation. It allows you to not repeat the same target if you want to call several methods on the same object.

List list = [];
list.add(color1);
list.add(color2);
list.add(color3);
list.add(color4);

// with cascade

List list = [];
list
  ..add(color1)
  ..add(color2)
  ..add(color3)
  ..add(color4);
